lighthouse-cli/test/smokehouse/offline-local/offline-expectations.js

```diff
@@ -65,7 +65,7 @@ module.exports = [
       },
       'user-timings': {
         score: 1,
-        displayValue: '',
+        displayValue: '0',
```

Comment:
this is just a result of audit.js stringifying `displayValue` by checking falsyness

https://github.com/GoogleChrome/lighthouse/blob/8fcbb8434220741e8e902cd48e18b1a325914003/lighthouse-core/audits/audit.js#L169

so `0` was getting converted to `''` before explicit stringifying in `user-timings.js` below